### PR TITLE
Setup test project and added Prometheus remote write client.

### LINKED ISSUE #3: Create new .NET unit test project

### LINKED ISSUE #6: Add RemoteWrite method to test project

### PR DESCRIPTION
**Pull request by RidgeWada**
Closes #3 
Closes #6

### CODE REVIEW

File: test/prom-query-tests/prom-query-tests.csproj [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <RootNamespace>prom_query_tests</RootNamespace>
```
Comment (sflanker) [inline test/prom-query-tests/prom-query-tests.csproj:5]: I suggest you make this `PrometheusQuerySdk.Tests`

Comment (sflanker) [reply test/prom-query-tests/prom-query-tests.csproj:5]: feel free to make that change in a future PR though

**Review by sflanker — APPROVED** [review]
Looks good.

**Comment by btakushi** [comment]
Looks good!

**Pull request merged**